docs/cloud/gcp.md

```diff
@@ -37,8 +37,63 @@ Configure BaGet to use GCS by updating the [`appsettings.json`](https://github.c
 
 ## Google Cloud SQL
 
-* TODO
+* Create a Google Cloud SQL instance (using MySQL 2nd Gen 5.7). The default options should work well.
+* Create a database named `baget`. This can be done through the Google Cloud Console. Use `utf8mb4` as the Character set.
```

Comment:
Do you have a link to documentation on how to use the console to make a database? That might be useful for someone that’s not familiar with GCP

Reply:
Added a link.